### ISSUE TITLE
Just got this up and running 

### ISSUE THREAD

**Issue by Tesla-P100FX** [mantorok1/homebridge-omnilink-platform#4]
**Describe Your Problem:**

Moved from HomeSeer with HomeSeer-Omni Bridge.   First of all its awesome.      I have a lot of zones in my house and run into an error that said i had more than a 155 zones.    So i am not including zones.   However i want to include my door switches so i know when a door is opened.   Can you tell me how to do that?    Also there is no ARM-Disarm showing in Homebridge.   

**Logs:**
/31/2021, 11:47:29 AM HOOBS listening on port 80.
3/31/2021, 11:47:31 AM Initializing HAP-NodeJS v0.9.3...
3/31/2021, 11:47:33 AM Loaded plugin 'homebridge-dyson-pure-cool'
3/31/2021, 11:47:33 AM [3/31/2021, 12:47:33 PM] Registering platform 'homebridge-dyson-pure-cool.DysonPureCoolPlatform'
3/31/2021, 11:47:33 AM Loaded plugin 'homebridge-myq'
3/31/2021, 11:47:33 AM [3/31/2021, 12:47:33 PM] Registering platform 'homebridge-myq.myQ'
3/31/2021, 11:47:33 AM Loaded plugin 'homebridge-omnilink-platform'
3/31/2021, 11:47:33 AM [3/31/2021, 12:47:33 PM] Registering platform 'homebridge-omnilink-platform.OmniLinkPlatform'
3/31/2021, 11:47:33 AM [3/31/2021, 12:47:33 PM] The plugin "homebridge-platform-wemo" requires Node.js version of ^14.16.0 which does not satisfy the current Node.js version of v14.15.1. You may need to upgrade your installation of Node.js - see https://git.io/JTKEF
3/31/2021, 11:47:33 AM Loaded plugin 'homebridge-platform-wemo'
3/31/2021, 11:47:33 AM Loading 4 platforms...
3/31/2021, 11:47:33 AM [3/31/2021, 12:47:33 PM] Registering platform 'homebridge-platform-wemo.BelkinWeMo'
3/31/2021, 11:47:33 AM [BelkinWeMo] Initializing BelkinWeMo platform...
3/31/2021, 11:47:33 AM [myQ] Initializing myQ platform...
3/31/2021, 11:47:33 AM [DysonPureCoolPlatform] Initializing DysonPureCoolPlatform platform...
3/31/2021, 11:47:33 AM [Omni] Initializing OmniLinkPlatform platform...
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: KitchenDesk
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: KitchenTable
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Stairway
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: xxxxx room
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Lxxxx Room
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: 2nd Flr.Hall
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Master Hall
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: MasterCloset
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Master Deck
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: UpperFloor
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Master Bed
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: O/SGarage
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Kitchen Bath
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Garage
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Basement Ctr
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: BasementO/S
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Kitchen Cntr
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Craft Cove
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Basem. Kitch
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Deck Chand
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Deck Sconce
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: MudRoom
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: KitchenCans
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: O/SMainE.LT
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Entry Chands
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Livingroom
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: LivingRmwest
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: DiningChand.
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: DiningWest
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: BackDeck
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: O/SN.Entry
3/31/2021, 11:47:33 AM [Omni] Configure unitswitch: Deck chandel
**Plugin Config:**

{
            "platform": "OmniLinkPlatform",
            "plugin_map": {
                "plugin_name": "homebridge-omnilink-platform"
            },
            "name": "Omni",
            "port": 4369,
            "includeAreas": false,
            "includeZones": false,
            "includeBypassZones": false,
            "includeButtons": true,
            "includeUnits": true,
            "includeThermostats": false,
            "includeEmergencyAlarms": false,
            "setHomeToAway": false,
            "setNightToAway": false,
            "syncTime": true,
            "showHomebridgeEvents": false,
            "showOmniEvents": true,
            "clearCache": false,
            "forceAutoDiscovery": false,
            "pushover": {
                "alarms": {
                    "burglary": false,
                    "fire": false,
                    "gas": false,
                    "auxiliary": false,
                    "freeze": false,
                    "water": false,
                    "duress": false,
                    "temperature": false
                },
                "troubles": {
                    "freeze": false,
                    "batterylow": false,
                    "acpower": false,
                    "phoneline": false,
                    "digitalcommunicator": false,
                    "fuse": false
                }
            },

**Screenshots:**

**Environment:**

* **Plugin Version**: 1.3.4
* **Homebridge Version**: <Hoobs 3.3.5>
* **Node.js Version**: <!14.15.1>
* **NPM Version**: 
* **Operating System**:

**Comment by Tesla-P100FX** [comment]
OK i turned on includeAreas to true and i can turn it on and off now.   Thank you!   i have Zones found: = 78.    So i don't want to show every zone in the home app, just really the doors.   in the log on Homebridge i see when a door is open.   I am looking to see in my Homekit app on my phone  that a door was opened.     The sensors are just the wired alarm switch's.   So do i need to include zones and then configure exceptions for all ones except the door?

**Comment by mantorok1** [comment]
I think the best option would be to set `includeZones` to `true` and then exclude the zones you don't want to see in HomeKit.  To do this use the `sensors` config option. For example, to excludes zone 4 & 6 you could use the following config:
```
    "sensors": [
      {
        "zoneId": 4,
        "sensorType": "none"
      },
      {
        "zoneId": 6,
        "sensorType": "none"
      }        
    ],
``` 
You can also use the above config to set your door switches to be a different type of sensor such as a "contact" instead of the default "motion" which may make more sense.

**Comment by Tesla-P100FX** [comment]
can I keep includeZones to no  and then jsu set the sensors i want to monitor as exceptions?  I have 80+ Sensors i don't want to have 80+ Contact switch's and i don't really want a line item for each if i can avoid it.  Or can i i put them all in one line like this?  

"zoneid": 4,5,6,7,8,9,10
"sensorType": "none"

or can i just add the ones i want with Include Zones set to  No 

"sensors": [
      {
        "zoneId": 4,
        "sensorType": "contact"
      },
      {
        "zoneId": 6,
        "sensorType": "contact"
      }        
    ],

**Comment by mantorok1** [comment]
Unfortunately you'll need to specify each zone individually and have includeZones set to true to do what you want. The only consolation is that this should just be once off config and then you never need to worry about it again. If you use copy & paste in a text editor it shouldn't take too long to get the config the way you want it.

I like the idea of assigning multiple zones IDs to a sensor type. I may incorporate that into a future version of the plugin.

**Comment by Tesla-P100FX** [comment]
Got it working!  Its really nice THANK YOU! I had to add 80 lines to the file , so i can just see our doors open and close and i get a notification on my phone.  I love it.   i also have a wired gate and i set it to turn on a light if the gate is opened at night.    In addition i was able to decommission my HomeSeer server

**Comment by mantorok1** [comment]
Hi @Tesla-P100FX ,

I've just released a new version of the plugin (1.4.0) which should make it easier to configure the plugin.
